angr/knowledge_plugins/cfg/cfg_model.py

```diff
@@ -99,13 +100,20 @@ def _get_cmsg(cls):
         return cfg_pb2.CFG()
 
     def serialize_to_cmessage(self):
+        if "Emulated" in self.ident:
+            l.error("Serializing a CFGEmulated instance is currently not supported.")
+            return None
+
         cmsg = self._get_cmsg()
         cmsg.ident = self.ident
 
         # nodes
         nodes = [ ]
         for n in self.graph.nodes():
-            nodes.append(n.serialize_to_cmessage())
+            n_cmsg = n.serialize_to_cmessage()
+            if n_cmsg is None:
```

Comment:
If a `NotImplementedError` is raised (as in my comment above), we don't need this check anymore.